python/pyspark/sql/functions.py

```diff
@@ -394,6 +394,23 @@ def cot(col):
     return _invoke_function_over_column("cot", col)
 
 
+def csc(col):
+    """
+    .. versionadded:: 3.3.0
```

Comment:
Shall we add a short distribution?

Reply:
Thank you, just added descriptions to all trig functions that missed descriptions.